### PR TITLE
Implemented todays time under user stats

### PR DESCRIPTION
**Pull request by ViddeM**
Solves #5

### CODE REVIEW

File: backend/src/schema/user.rs [hunk 1]
```diff
@@ -187,6 +187,38 @@ impl User {
...
+    let today_end = today_start
+      .checked_add_signed(Duration::days(1))
+      .ok_or(HubbitSchemaError::InternalError)?
+      .checked_sub_signed(Duration::seconds(1))
+      .ok_or(HubbitSchemaError::InternalError)?;
+
+    let sessions = user_session_repo
+      .get_range_for_user(today_start, today_end, self.id)
+      .await
+      .map_err(|_| HubbitSchemaError::InternalError)?;
```
Comment (rosengrenen) [inline backend/src/schema/user.rs:214]: Should probably log an error in the console

File: frontend/src/dateUtil.ts [hunk 2]
```diff
@@ -82,3 +82,28 @@ export function prettyFromSeconds(seconds: number) {
 
   return pretty;
 }
+
+export function getHoursDiff(a: Date, b: Date): string {
```
Comment (rosengrenen) [inline frontend/src/dateUtil.ts:86]: Do we care for singular and plural?

**Comment by ViddeM** [comment]
Don't merge yet, should change backend to just use the add / sub without checked stuff.

**Review by rosengrenen — CHANGES_REQUESTED** [review]

File: backend/src/schema/user.rs (hunk 1 above)
Comment (ViddeM) [reply backend/src/schema/user.rs:214]: Feck

File: frontend/src/dateUtil.ts (hunk 2 above)
Comment (ViddeM) [reply frontend/src/dateUtil.ts:86]: True

Comment (ViddeM) [reply frontend/src/dateUtil.ts:86]: Fixed

**Pull request merged**